src/Skills/__tests__/Skills.test.js

```diff
@@ -1,25 +1,81 @@
 import React from 'react';
-import Skill from '../../Skill/Skill';
-import Skills from '../Skills';
-import { mount } from 'enzyme';
+import { Skills, GET_SKILLS_QUERY } from '../Skills';
+import { act, render } from '@testing-library/react';
+import { MockedProvider } from '@apollo/react-testing';
+const wait = require('waait');
 
-test('renders available skills heading', () => {
-  const wrapper = mount(<Skills/>);
+const mock = {
+    request: {
+      query: GET_SKILLS_QUERY
+    },
+    result: {
+      data: {
+        allSkills: [{ id: '1', title: 'Title', description: 'Description' }],
+      },
+    },
+  };
 
-  expect(wrapper.contains(<h1>Available Skills</h1>)).toEqual(true);
+let wrapper;
+
+test('renders without error', () => {
+  act(() => {
```

Comment:
Nitpick: these three `act` calls are all similar (they render a `MockedProvider` with `Skills` inside) - is it worth extracting out a helper method to avoid repetition/make it quicker to change?

Reply:
Yep I agree - will address in [Loofah-13](https://devservices.jira.com/browse/LOOFAH-13) to avoid conflicts